frontend/js/src/playlists/Playlist.tsx

```diff
@@ -392,6 +377,10 @@ export default class PlaylistPage extends React.Component<
     const collaboratorsWithoutOwner = collaborators.filter(
       (username) => username.toLowerCase() !== playlist.creator.toLowerCase()
     );
+    if (isPlaylistOwner(playlist, currentUser)) {
+      this.alertNotAuthorized();
+      return;
+    }
```

Comment:
This looks like it does the inverse to what was happening previously on line 381 (`if (!this.isOwner())`)

Reply:
Yes, it must be a typing error I have corrected it.